substrate/frame/child-bounties/src/benchmarking.rs

```diff
@@ -105,7 +99,7 @@ fn activate_bounty<T: Config>(
 ) -> Result<BenchmarkChildBounty<T>, BenchmarkError> {
 	let mut child_bounty_setup = setup_child_bounty::<T>(user, description);
 	let curator_lookup = T::Lookup::unlookup(child_bounty_setup.curator.clone());
-	Bounties::<T>::propose_bounty(
+	pallet_bounties::Pallet::<T>::propose_bounty(
```

Comment:
Not wrong, but just asking: why the renames?

Reply:
Hmm, just syncing style with other benchmarks.

I also found that there're some `frame_system::Pallet` in this file. I think this is more clear?

But I'm ok with both style.